pkg/domain/config.go

```diff
@@ -0,0 +1,40 @@
+package domain
+
+import (
+	"context"
+)
+
+// PostgresConfig contains the Postgres database configuration arguments
+type PostgresConfig struct {
+	Hostname     string `json:"hostname"`
```

Comment:
I don't think the json tags are correct here. Looking at the [docs](https://github.com/asecurityteam/settings), it looks like this should be a description tag with a short description of each variable.

Reply:
That's probably me being lazy.. I don't think we need json here... at least not yet. I'll take them out

Reply:
I think what @zlozano is saying is that `json` tags _won't_ be used here since this only interfaces with the settings loader and it has a different set of tags for adding metadata.